SonarQube.TeamBuild.PreProcessor/AnalysisConfigGenerator.cs

```diff
@@ -48,12 +48,14 @@ public static class AnalysisConfigGenerator
             config.SonarProjectVersion = args.ProjectVersion;
             config.SonarQubeHostUrl = args.GetSetting(SonarProperties.HostUrl);
 
-            config.SetBuildUri(settings.BuildUri);
-            config.SetTfsUri(settings.TfsUri);
-            config.SonarConfigDir = settings.SonarConfigDirectory;
-            config.SonarOutputDir = settings.SonarOutputDirectory;
-            config.SonarBinDir = settings.SonarBinDirectory;
-            config.SonarRunnerWorkingDirectory = settings.SonarRunnerWorkingDirectory;
+            config.SetBuildUri(buildSettings.BuildUri);
+            config.SetTfsUri(buildSettings.TfsUri);
```

Comment:
The shim is a simple wrapper around the Java sonar-runner. It shouldn't have dependencies on anything except Common and shouldn't know anything about TFS.